common/build.gradle

```diff
@@ -1,4 +1,3 @@
-group "com.mx.path.core"
 jar.archiveBaseName = "common"
```

Comment:
I was looking at 'archiveBaseName' yesterday because my IntelliJ says it's read-only. I'm not sure but it seems like we may be trying to set the wrong thing. Are we sure it's not archiveFileName that we're wanting to set?

https://docs.gradle.org/current/dsl/org.gradle.api.tasks.bundling.Jar.html

Reply:
Yeah, that's one thing I didn't really dig into. I don't think setting `jar.archiveBaseName` is correct, but I didn't want to change it without researching the proper way to handle it.